templates/index.html

```diff
@@ -1054,35 +1054,41 @@ <h4 class="modal-title">העדפות:</h4>
 
 <script type="text/javascript">
 
-            {% if minimal_zoom %}
-            MINIMAL_ZOOM = {{minimal_zoom}};
+            {% if CONST %}
+                {% for key,val in CONST.iteritems() %}
+                   {{key}} = {{val}}
+                {% endfor %}
             {% else %}
-            MINIMAL_ZOOM = 16;
+            console.error("CONST doesn't exists, failed to share constants between python and javascript");
             {% endif %}
-            {% if marker_type_accident %}
-            MARKER_TYPE_ACCIDENT = {{marker_type_accident}};
-            {% else %}
-            MARKER_TYPE_ACCIDENT = 1;
+
+            {% if not 'MINIMAL_ZOOM' in CONST %}
```

Comment:
the format if not, is used in project. example:
 if not SESSION_HIGHLIGHTPOINT_KEY in session: (main.py)